src/components/MoviesList/MoviesList.scss

```diff
@@ -1 +1,10 @@
-// 
+.page {
```

Comment:
why you moved these styles from `App.scss` if `page` and `page-content` still in `App.tsx`?